holoviews/core/data/multipath.py

```diff
@@ -0,0 +1,160 @@
+import numpy as np
+
+from ..util import max_range
+from .interface import Interface
+
+class MultiInterface(Interface):
+    """
+    MultiInterface allows wrapping around a list of tabular datasets
+    including dataframes, the columnar dictionary format or 2D tabular
+    NumPy arrays. Using the split method the list of tabular data can
+    be split into individual datasets.
+    """
+
+    types = ()
+
+    datatype = 'multi'
```

Comment:
Wouldn't mind a longer, more descriptive name but can't think of a good suggestion right now.